### PR TITLE
Creating debian packages

### PR DESCRIPTION
**Pull request by shubhajeet**
**Trying to create debian package of batti**.
Hoping to include it in our own repo in foss-np/deb-packages (fosspulchowk/deb-packages)in few weeks.

*Note: I am getting bad quality package when trying to install the package but everything else is working fine*

### CODE REVIEW

**Comment by rhoit** [comment]
@shubhajeet please use the `dev` repo.. for new commits!

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 SOURCES = $(wildcard *.sh)
-SUPPORT = README.org AUTHORS LICENSE .version
+SUPPORT = README.md AUTHORS LICENSE .version
```
Comment (rhoit) [inline Makefile]: We are phasing out the `markdown` and moving to `emacs` `org-mode`.

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +0,0 @@
-[submodule "2utf8"]
```
Comment (rhoit) [inline .gitmodules]: We already cleaned up these things in `dev` branch.

File: Makefile [hunk 3]
```diff
@@ -35,22 +35,22 @@ installcheck:
...
 
-droid:
-	@echo "# Compacting for Android"
+dist-android:
+	@echo "Compacting for Android"
 	mkdir droid
-	sed 's|HOME/.cache|WD|' main.sh > droid/batti.sh
+	sed 's/.HOME.*\///' main.sh > droid/batti.sh
 	chmod +x droid/batti.sh
-	./main.sh -d > droid/batti.sch
```
Comment (rhoit) [inline Makefile]: It seems like you are not *syncing* your *repo*. You got the merge the new changes before doing further work.
If you have trouble handeling multiple `remote` `repos`, I have given you access to the repository. Create `dev-sujit` branch and directly work on it. 

# WHY I'M SAYING THIS?

`master` *should* be stable branch, we test in `dev` and *push* it to `master`.

**Comment by rhoit** [comment]
Closing this request will be pulled from `dev`.

**Comment by shubhajeet** [comment]
@rhoit is emacs org mode supported by github

**Comment by rhoit** [comment]
@shubhajeet yes, you can check it in `dev` branch.

File: Makefile (hunk 1 above)
Comment (munikarmanish) [inline Makefile]: Why not markdown? @rhoit

Comment (rhoit) [inline Makefile]: coz i'm maintaing it. :joy:

**Pull request closed without merging**